versions.mk

```diff
@@ -14,7 +14,7 @@
 
 LIB_NAME := nvidia-container-toolkit
 LIB_VERSION := 1.15.0
-LIB_TAG := rc.3
+LIB_TAG := rc.4
```

Comment:
Nit: Could we have the version bump as a separate commit?

Reply:
Sounds good, I just split this into two commits. Thanks!